containers/ldap.docker/makecerts.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+# CA, server and client key pairs:
+
+openssl genpkey -algorithm rsa -pkeyopt rsa_keygen_bits:4096 -out ca.key
+openssl genpkey -algorithm rsa -pkeyopt rsa_keygen_bits:4096 -out server.key
+openssl genpkey -algorithm rsa -pkeyopt rsa_keygen_bits:4096 -out client.key
+
+# Create the X509 certificate for the CA:
+
+openssl req -new -key ca.key -x509 -days 1095 -subj "/C=DE/ST=NRW/L=Cologne/O=ArangoDB GmbH/OU=Clifton/CN=MaxNeunhoeffer/emailAddress=max@arangodb.com/" -out ca.pem
+
+# Create a certificate signing request for the server and client certificates:
+
+openssl req -new -key server.key -subj "/C=DE/ST=NRW/L=Cologne/O=ArangoDB GmbH/OU=Clifton/CN=ldap01.arangodb.biz/emailAddress=max@arangodb.com/" -out server.csr
+openssl req -new -key client.key -subj "/C=DE/ST=NRW/L=Cologne/O=ArangoDB GmbH/OU=Clifton/CN=ldap01.arangodb.biz/emailAddress=max@arangodb.com/" -out client.csr
```

Comment:
Should we use @neunhoef here? Maybe something neutral? Also do we really have `ldap01.arangodb.biz`?

Reply:
yes